thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/notification/formatter/channels/AlertContentFormatter.java

```diff
@@ -0,0 +1,41 @@
+package org.apache.pinot.thirdeye.notification.formatter.channels;
+
+import java.util.Properties;
+import org.apache.pinot.thirdeye.anomaly.ThirdEyeAnomalyConfiguration;
+import org.apache.pinot.thirdeye.datalayer.pojo.AlertConfigBean;
+import org.apache.pinot.thirdeye.notification.content.BaseNotificationContent;
+import org.apache.pinot.thirdeye.notification.formatter.ADContentFormatterContext;
+
+
```

Comment:
Maybe add some documentation for this class? What's the functionality for the class and how it should be used.

Reply:
added